### PR TITLE
add global referrence style links to balenaHub

### LINKED ISSUE #1812: Add generic global link to balenaHub
> @vipulgupta2048: Add balenaHub to links.coffee file for it to be referenced later - https://github.com/balena-io/docs/blob/master/config/links.coffee I assume we would like to add balenaHub a lot of times throughout the docs 
>
> Adding it to links module helps us refer to it consistently all over the place. Check this for reference https://github.com/balena-io/docs/blob/790bb6ebc1640a8526a9672547286bf866476d1e/CONTRIBUTING.md#adding-links-to-the-docs

We also need to track down all the other hard coded links and replace them with the generic one

and more things  https://github.com/balena-io/docs/issues/1812#issuecomment-814090205

### PR DESCRIPTION
**Pull request by somombo**
Closes #1812

Signed-off-by: Somo S <somo@balena.io>

### CODE REVIEW

**Comment by somombo** [comment]
not sure if I was supposed to use minor change type @vipulgupta2048

File: pages/learn/getting-started.md [hunk 1]
```diff
@@ -177,11 +177,11 @@ Your application will then be downloaded and executed by all the devices you hav
 
 ## Next steps
 
-- To easily find, deploy and publish your applications for {{ $names.company.lower }} [supported devices][supported-devices], head to [balenaHub](https://hub.balena.io)
+- Head to [balenaHub][balenahub], for more ready-to-deploy applications, or have a go at building and sharing your own cool projects using [balenaBlocks][balenablocks].
 - Try out [local mode][local-mode], which allows you to build and sync code to your device locally for rapid development.
 - Develop an application with [multiple containers][multicontainer] to provide a more modular approach to application management.
 - Manage your device fleet with the use of [configuration][configuration], [environment][service], and [service variables][service].
-- Explore these [example projects][projects] to give you an idea of more things you can do with {{ $names.company.lower }}.
+- Visit our blog to find step-by-step tutorials for some [classic balena projects][projects]
```
Comment (vipulgupta2048) [inline pages/learn/getting-started.md:184]:
```suggestion
- Visit our blog to find step-by-step tutorials for some [classic balena projects][projects].
```

**Review by vipulgupta2048 — COMMENTED** [review]
It seems a new package-lock.json got added for some reason in your commits, can you resolve that. Thanks! 
Regarding your question of change-type, I think this could be patch. Just a recommendation.

File: pages/learn/getting-started.md (hunk 1 above)
Comment (vipulgupta2048) [reply pages/learn/getting-started.md:184]: nitpick

**Comment by somombo** [comment]
> It seems a new package-lock.json got added for some reason in your commits, can you remove that. Thanks!
> Regarding your question of change-type, I think this could be patch. Just a recommendation.

@vipulgupta2048 fixed

**Review by vipulgupta2048 — APPROVED** [review]

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Pull request merged**